hadoop/yarn/yarn_mutation.py

```diff
@@ -2,17 +2,38 @@
 from xml.etree import ElementTree as ET
 
 
-class YarnMutationConfig:
+class MutationBase(object):
```

Comment:
Instead of a base class and separate classes, I think we could use a simple mutation config class, that is able to handle multiple types of mutations eg. add-queue, update-queue etc.. within the common sched-conf tag. It would look like:
```python
mutation.add_queue(queue, {})
mutation.update_queue(queue, {})
mutation.remove_queue(queue, {})
```

Reply:
Done.